cmd/kube-apiserver/app/server.go

```diff
@@ -176,6 +176,11 @@ func Run(completeOptions completedServerRunOptions, stopCh <-chan struct{}) erro
 	return prepared.Run(stopCh)
 }
 
+// LastKubeAPIServerConfigSeenInTest allows tests to make assertions about the Kubernetes API server config.
+// Tests using this variable must run serially as this variable is not safe to read concurrently with server start.
+// Tests must never mutate this variable.
+var LastKubeAPIServerConfigSeenInTest *controlplane.Config
```

Comment:
*raises eyebrow*

is there an alternate way to figure out if we're constructing multiple instances of the storage transformer in a test that doesn't make the production code set test-only vars? like making the storage transformer constructer a var that test can swap/wrap to instrument number of constructions or something?

Reply:
Yeah I can do the number of calls to the constructor as a proxy to test this.